Buildings/Fluid/CHPs/BaseClasses/AssertFuelFlow.mo

```diff
@@ -0,0 +1,94 @@
+within Buildings.Fluid.CHPs.BaseClasses;
+model AssertFuelFlow "Assert if fuel flow is outside boundaries"
+  extends Modelica.Blocks.Icons.Block;
+
+  parameter Boolean dmFueLim
+    "If true, the rate at which fuel mass flow rate can change is limited";
```

Comment:
Simply conditionally remove the whole block if the mass flow rate should not be limited. This will be much less overhead.